### PR TITLE
Feature/master/non csv feeder

### PR DESCRIPTION
**Pull request by cprice404**
This commit introduces a feeder class in scala code, so that we don't need CSV files for all of the various numbers of feeders that we might want to use.

I think that it might break any existing sims that are explicitly using feeders via csv, but not 100% sure whether that is true, or whether I care :)

### CODE REVIEW

File: simulation-runner/src/main/scala/com/puppetlabs/gatling/runner/ConfigDrivenSimulation.scala [hunk 1]
```diff
@@ -101,7 +101,10 @@ class ConfigDrivenSimulation extends Simulation {
     val chainWithSleeps:ChainBuilder =
       addSleeps(chainWithFailFast, sleepDuration, numRepetitions)
 
+    val feeder = NodeFeeder("node", numInstances).circular
```
Comment (haus) [inline simulation-runner/src/main/scala/com/puppetlabs/gatling/runner/ConfigDrivenSimulation.scala]: `num_instances` from a scenario gets converted into `numInstances` here somewhere along the way?

Comment (haus) [inline simulation-runner/src/main/scala/com/puppetlabs/gatling/runner/ConfigDrivenSimulation.scala]: ah. found it. https://github.com/puppetlabs/gatling-puppet-load-test/blob/34f30e164580933d6f216d56f878ed2fb193c987/simulation-runner/src/main/scala/com/puppetlabs/gatling/config/PuppetGatlingConfig.scala#L23

**Comment by haus** [comment]
this is awesome. :100:

**Comment by jpinsonault** [comment]
Very cool :+1:

**Pull request merged**